src/Sentry/SentryOptionsExtensions.cs

```diff
@@ -87,9 +87,16 @@ public static void DisableNetFxInstallationsIntegration(this SentryOptions optio
         /// <param name="options">The SentryOptions to hold the processor.</param>
         /// <param name="integration">The integration.</param>
         public static void AddIntegration(this SentryOptions options, ISdkIntegration integration)
-            => options.Integrations = options.Integrations != null
-                ? options.Integrations.Concat(new[] { integration }).ToArray()
-                : new[] { integration };
+        {
+            if (options.Integrations == null)
+            {
+                options.Integrations = new[] {integration};
+            }
+            else
+            {
+                options.Integrations = options.Integrations.Concat(new[] {integration}).ToArray();
```

Comment:
Just a thought, but maybe we should leave `AddDiagnosticSourceIntegration` as it was, and check here more generally instead?   I mean, can we safely say that there should not be more than one integration of the same type?

Reply:
@mattjohnsonpint i asked @bruno-garcia the same question. he said it is valid to have more than one integration by type, but with different settings in each integration instance

Reply:
Where did that communication happen?

The example was: `new FileWatcherIntegration("path1"), new FileWatcherIntegratoin("path2")`

Reply:
LGTM